ui/app/AppLayouts/Communities/views/CommunityTokenView.qml

```diff
@@ -228,7 +226,7 @@ StatusScrollView {
             id: tokenHolderLoader
 
             visible: !root.preview && root.deploymentCompleted
-            sourceComponent: isOwnerTokenItem ? tokenHolderContact : (token.tokenHoldersLoading ? tokenHoldersLoadingComponent : sortableTokenHolderPanel)
```

Comment:
Can `isOwnerTokenItem` property be removed then?

Reply:
By removing `tokenHolderContanct` we are removing as well this placeholder. Do we want to? 
![Screenshot 2025-05-27 at 13 23 42](https://github.com/user-attachments/assets/600a0095-be87-498b-bf2d-20f13f8dede9)

Reply:
Yes it's ok, because `sortableTokenHolderPanel` also has a similar panel.